script/bootstrap.py

```diff
@@ -42,7 +42,7 @@ def main():
 
   # Redirect to use local libchromiumcontent build.
   if args.build_release_libcc or args.build_debug_libcc:
-    build_libchromiumcontent(args.verbose, args.target_arch, defines,
```

Comment:
Are `defines` used anywhere else?

Reply:
I may not understand the question, but it looks like defines are passed a little further down in this file to electron:script/update.py where it uses them in its call to gyp.

Reply:
Yeah, you're right. Sorry it took so long.